adobe/react-spectrum
# Tableview crash bug

Might be related to https://github.com/adobe/react-spectrum/issues/5357

Some observations:
- Throttling the cpu seems to help prevent the crash
- Virtualizer is bouncing between rects
- Implementing a fix like the one here https://github.com/adobe/react-spectrum/pull/4981/files (with some minor changes) does help to solve the issue

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Pull down branch
2. Run storybook locally
3. Go to TableView > Async Reload on Sort
4. Set window to 1082.73 width (or as close to it as possible) 90% zoom
5. Try sorting  the `Title` column by clicking on it
6. It should crash but you might need to click on `Title` a couple times to get it to work. See if you notice any difference with Strictmode on or off. 

Note: It is a bit finicky but both Rob and I have been able to reproduce it at this width and zoom pretty consistently. 


## 🧢 Your Project:
